azure-pipelines.yml

```diff
@@ -423,7 +423,7 @@ jobs:
       targetType: 'filePath'
       filePath: ./scripts/ci/test_automation.sh
     env:
-      REDUCE_SDK: 'True'
+      ADO_PULL_REQUEST_TARGET_BRANCH: $(System.PullRequest.TargetBranch)
```

Comment:
why do you add this environment ADO_PULL_REQUEST_TARGET_BRANCH

Reply:
Because in scripts (both in .sh and .py) can't reference ADO pre-defined variables. So I have to add it here.

Also, I enable incremental test on all target branch instead of dev only on PR stage. This is also another keypoint I set this ADO variable environment.